x/validators/handler_test.go

```diff
@@ -35,8 +38,44 @@ func TestHandler(t *testing.T) {
 				PubKey: abci.PubKey{Data: alice.PublicKey().GetEd25519(), Type: "ed25519"},
 				Power:  10,
 			}},
+			DbExp: ValidatorUpdates{[]ValidatorUpdate{{Pubkey: Pubkey{Data: bobby.PublicKey().GetEd25519(), Type: "ed25519"}, Power: 3}, {
+				Pubkey: Pubkey{Data: alice.PublicKey().GetEd25519(), Type: "ed25519"},
+				Power:  10,
+			}}},
+		},
+		"Adding a validator works with pre-exiting one": {
+			Src: []*ValidatorUpdate{{
+				Pubkey: Pubkey{Data: alice.PublicKey().GetEd25519(), Type: "ed25519"},
+				Power:  10,
+			}},
+			AuthzAddress: alice.PublicKey().Address(),
+			Exp: []abci.ValidatorUpdate{{
+				PubKey: abci.PubKey{Data: alice.PublicKey().GetEd25519(), Type: "ed25519"},
+				Power:  10,
+			}},
+			DbExp: ValidatorUpdates{[]ValidatorUpdate{{
+				Pubkey: Pubkey{Data: alice.PublicKey().GetEd25519(), Type: "ed25519"},
+				Power:  10,
+			}}},
+		},
+		"Setting different power is allowed": {
+			Initial: ValidatorUpdates{ValidatorUpdates: []ValidatorUpdate{{Pubkey: Pubkey{Data: alice.PublicKey().GetEd25519(), Type: "ed25519"}, Power: 3}}},
```

Comment:
I think so too.

How about we refactor this to be weave default and only convert on deliver?
I'm going to do that.

Reply:
Looking at this now, I have moved ValidatorUpdate to weave and upgraded everything to use that. But we still define `ValidatorUpdates` in `x/validators` because it only makes sense in it's context when we store stuff there. 
I guess the next good step would be to make it work on weave level, but I don't want to do it in this PR.